src/lib/shared/ui/optimal-image/tailwind-weserv-interop.ts

```diff
@@ -0,0 +1,61 @@
+const width = /\bw-([0-9]+)\b/;
+const height = /\bh-([0-9]+)\b/;
+
+interface ImageDimensions {
+  width: number | undefined;
+  height: number | undefined;
+}
+
+export function parseTailwindSize(classes: string) {
```

Comment:
Suggestion (non-blocking): It'd be nice if you provided comments for the functions in this file